migrations/2021-04-16-130927_language-tags/up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE comment ADD COLUMN language TEXT NOT NULL;
+ALTER TABLE post ADD COLUMN language TEXT NOT NULL;
+ALTER TABLE private_message ADD COLUMN language TEXT NOT NULL;
+
+ALTER TABLE local_user ADD COLUMN discussion_languages TEXT[] NOT NULL;
```

Comment:
Big SQL no-no, never put a list into a column. This needs to be a separate table with `(id,  local_user_id, language)`